### PR TITLE
fix(files): fix html structure on the profile page

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
## Summary

Before a button was mistakenly inside of the heading. Now all headings are standalone headings and have no buttons and other interactive content inside.

## Screenshots

Before | After
---|---
image | Screenshot from 2024-02-28 16-42-24

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/PersonalInfo/AvatarSection.vue [hunk 1]
```diff
@@ -275,6 +275,7 @@ export default {
 <style lang="scss" scoped>
 section {
 	grid-row: 1/3;
+  padding: 10px 10px;
```
Comment (susnux) [inline apps/settings/src/components/PersonalInfo/AvatarSection.vue:278]: wrong indention

**Review by susnux — APPROVED** [review]
Except the wrong indention (I suspect wrong tab width) this looks fine

**Review by ShGKme — APPROVED** [review]

File: apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue [hunk 2]
```diff
@@ -21,11 +21,11 @@
 -->
 
 <template>
-	<component :is="isHeading ? `h3` : `div`" class="headerbar-label" :class="{ 'setting-property': isSettingProperty, 'profile-property': isProfileProperty }">
-		<span v-if="isHeading">
+	<div class="headerbar-label" :class="{ 'setting-property': isSettingProperty, 'profile-property': isProfileProperty }">
+		<h3 v-if="isHeading">
```
Comment (Pytal) [inline apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue:25]: We should remove the h3 `Your profile information` then as it doesn't seem to make sense in the hierarchy and there is already an h2 `Personal info`

**Review by Pytal — DISMISSED** [review]

File: apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue (hunk 2 above)
Comment (susnux) [reply apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue:25]: Yes the duplicated heading should be removed, so the hidden "Your profile information".

Comment (JuliaKirschenheuter) [reply apps/settings/src/components/PersonalInfo/shared/HeaderBar.vue:25]: done

**Review by emoral435 — APPROVED** [review]
LGTM :)

**Pull request merged**